docs/api.rst

```diff
@@ -72,19 +72,28 @@ that are not directly implemented :mod:`biotite.structure`.
     fnat
     DockQ
 
+Atom Matching
```

Comment:
Just curious what your reasoning is behind not using something like `sphinx-apidoc`?

Reply:
`sphinx-apidoc` is quite inflexible, e.g. it would not allow to emphasize the `Evaluator` and to define the custom rubrics.